win32.go

```diff
@@ -0,0 +1,121 @@
+// +build windows
+
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"syscall"
+	"unsafe"
+
+	"github.com/mitchellh/go-ps"
+)
+
+type win32Info interface {
```

Comment:
As this is now part of `environment`, the interface here is redundant. I can live with it being removed as mocking is possible via `MockedEnviroment` anyhow.